packages/suite/src/middlewares/wallet/storageMiddleware.ts

```diff
@@ -197,16 +198,19 @@ const storageMiddleware = (api: MiddlewareAPI<Dispatch, AppState>) => {
                     }
                     break;
                 }
-                case SEND.STORE_DRAFT: {
+                case sendFormActions.storeDraft.type: {
```

Comment:
If action creator is used, move it outside of switch / case and use action matcher please.

Reply:
fixed by b2b40263ba